mcp-server/test_mcp_mocked.py

```
@@ -0,0 +1,243 @@
+#!/usr/bin/env python3
+
+import pytest
+from unittest.mock import AsyncMock, MagicMock, patch
```

Ruff (0.11.9)

`unittest.mock.MagicMock` imported but unused

Remove unused import: `unittest.mock.MagicMock`

(F401)